fdsc/fdsc.user.js

```diff
@@ -23,14 +24,17 @@
 // @exclude     *://blog.stackoverflow.com/*
 // @exclude     *://*.area51.stackexchange.com/*
 // @require     https://cdn.rawgit.com/ofirdagan/cross-domain-local-storage/d779a81a6383475a1bf88595a98b10a8bd5bb4ae/dist/scripts/xdLocalStorage.min.js
+// @require     https://wzrd.in/standalone/debug@%5E2.6.6
 // @grant       none
 // ==/UserScript==
 
-/* global fdsc, $, xdLocalStorage, confirm */
+/* global fdsc, $, xdLocalStorage, confirm, unsafeWindow */
 /* eslint-disable max-nested-callbacks */
 
 (function () {
   "use strict";
+  const createDebug = typeof unsafeWindow === "undefined" ? window.debug : unsafeWindow.debug || window.debug;
```

Comment:
Wouldn't this be functionally equal, but cleaner?

    const createDebug = (unsafeWindow || window).debug;